script/spec-runner.js

```diff
@@ -216,7 +216,7 @@ async function installSpecModules (dir) {
     env.npm_config_nodedir = path.resolve(BASE, `out/${utils.getOutDir({ shouldLog: true })}/gen/node_headers`);
   }
   if (fs.existsSync(path.resolve(dir, 'node_modules'))) {
-    await fs.remove(path.resolve(dir, 'node_modules'));
+    fs.rmSync(path.resolve(dir, 'node_modules'), { force: true, recursive: true });
```

Comment:
Why the change away from async here?

Reply:
Here's a learning opportunity for me -- is there a difference between calling `await fs.promises.rm()` and `fs.rmSync()`? e.g. is one API more performant or more stable?

I assumed that sync calls were OK because the previous line uses `fs.existsSync()`, but maybe that's a special case because `fs.exists()` is deprecated?

Reply:
I'm of two minds on this.

OT1H, I agree with your comment that a procedural PR like this should KISS, e.g. avoid the refactors in `gn-asar`. IMO this outweighs the rest of the discussion. I'll revise to reduce diffs from `main`.

OT2H, this seems like a learning opportunity for me, so I have to ask -- is there a strong reason to preserve this `await fs.promises.foo()` call instead of calling `fs.fooSync()` when the pre-existing previous line is also making an fs sync call?

Reply:
@ckerr Because the function this is encapsulated in is async (async function installSpecModules), I'd lean towards using async, but the fact that there is already sync calls mixed in it doesn't make too much difference other than keeping the the updated code working the same way that the previous code did.